definitions/tools/filter_sv_vcf.cwl

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+
+baseCommand: ["/bin/bash","filter_sv_vcf_read_support.sh"]
+requirements:
+    - class: ResourceRequirement
+      ramMin: 4000
+    - class: DockerRequirement
+      dockerPull: "mgibio/bcftools-cwl:1.9"
+    - class: InitialWorkDirRequirement
+      listing:
+      - entryname: "filter_sv_vcf_read_support.sh"
+        entry: |
+          set -eou pipefail
```

Comment:
Not strictly necessary, but might be nice to have a `#!/bin/bash` line atop this file.

Reply:
done!